react-regex.js

```diff
@@ -174,5 +174,15 @@ ReactRegex = [
   {
     regex: /\stabindex=/g,
     replace: " tabIndex="
+  },
+  // Template.contentBlock
+  {
+    regex: /({{>\s*Template.contentBlock\s*}})/g,
+    replace: "{this.props.children}"
+  },
+  // @index -- match @index by itself {{@index}} or as a param to helper {{helper @index anotherParam}}
+  {
+    regex: /({{.*@index.*}})/g,
+    replace: "{index}"
```

Comment:
yea, the way you're injecting variables such as `context` (and `index` from `each in`) into scope will make this work as long as something else later injected doesnt override the scope or variable. 

ps. we need to get some support on this project. Look, you were discussed in depth in Josh Owen's latest podcast: https://www.youtube.com/watch?v=BCNEmB3mI78 ...It was near the beginning (within the first 15 I recall). Sideburns was literally one of the 3-4 pre-scheduled segments on the show. This is a hot project. Lots of people are gonna benefit from it. We need to find a way to get some of these guys to help. I know there are a few guys that will pick it up and run with it hard within their own packages. So they should be helping us so we can get this done quicker.